src/steps/ms-defender/finding/converters.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  createIntegrationEntity,
+  createDirectRelationship,
+  Entity,
+  Relationship,
+  RelationshipClass,
+  createMappedRelationship,
+  getTime,
+} from '@jupiterone/integration-sdk-core';
+import { Finding } from '../../../types';
+import { entities, MappedRelationships, TargetEntities } from '../constants';
+
+const SEVERITY_TO_NUMERIC_SEVERITY_MAP = new Map<string, number>([
+  ['low', 2],
+  ['medium', 5],
+  ['high', 7],
+  ['critical', 10],
+]);
+
+export function getNumericSeverityFromIssueSeverity(
+  issueSeverity?: 'low' | 'medium' | 'high' | 'critical',
+): number {
+  if (!issueSeverity) return 0;
+
+  const numericSeverity = SEVERITY_TO_NUMERIC_SEVERITY_MAP.get(issueSeverity);
+  return numericSeverity === undefined ? 0 : numericSeverity;
+}
+
+export function createFindingEntity(
+  defenderFinding: Finding,
+  machineEntity,
+): Entity {
+  return createIntegrationEntity({
+    entityData: {
+      source: {},
+      assign: {
+        id: defenderFinding.id,
+        _type: entities.FINDING._type,
+        _class: entities.FINDING._class,
+        _key: machineEntity.id + '_' + defenderFinding.id,
+        name: defenderFinding.name,
+        displayName: defenderFinding.name,
+        description: defenderFinding.description,
+        severity: defenderFinding.severity,
+        score: defenderFinding.cvssV3,
+        numericSeverity: getNumericSeverityFromIssueSeverity(
+          defenderFinding.severity.toLowerCase(),
+        ),
+        publishedOn: defenderFinding.publishedOn,
+        updatedOn: getTime(defenderFinding.updatedOn),
+        category: '',
+        open: true,
```

Comment:
Is it the case that all findings returned from the API are essentially open vulnerabilities? Is there any indication in the response data that a vulnerability has been addressed and is now closed?